wiki/Glossary/Cookie/en.md

```diff
@@ -0,0 +1,16 @@
+---
+tags:
+  - logo
+---
+
+# Cookie
+
+*For cookie usage guidelines, see: [Brand identity guidelines](/wiki/Brand_identity_guidelines)*
+
+*Cookie* is another word for the osu! logo and refers to its shape. The cookie appears many times in the osu! client. It can be seen in the main menu, solo mode, and previously in old crash reports as seen below.
```

Comment:
```suggestion
**Cookie** is another word for the osu! logo and refers to its shape. The cookie appears many times in the osu! client. It can be seen in the main menu, solo mode, and previously in old crash reports as seen below.
```